test/param/number-list.spec.js

```diff
@@ -41,4 +41,54 @@ describe('Testing NumberList Parameter', () => {
       }
     })).to.throw('Invalid Value for json-Parameter "list" provided.');
   });
+
+  describe('Testing optional parameter "minItemValue"', () => {
```

Comment:
I would have a third test showing that if all items in the list are larger than `minItemValue`, you will not get an error.